docs/getting-started.md

```diff
@@ -133,6 +133,14 @@ You will need Node, the React Native command line interface, a JDK, and Android
 
 <block class="native windows android" />
 
+<h2> Intalling React Native Cli</h2>
+
+We recommend intalling Ract Native Cli
+
+```sh
+sudo npm install -g react-native-cli
```

Comment:
You have added the section in the Android & Windows guide, without a WSL there is no Bash (and `sudo`) on Windows.